config.json

```diff
@@ -156,48 +156,48 @@
       {
         "groupId": "com.google.android.gms",
         "artifactId": "play-services-awareness",
-        "version": "19.0.1",
-        "nugetVersion": "119.0.1.11",
+        "version": "19.1.0",
+        "nugetVersion": "119.1.0",
         "nugetId": "Xamarin.GooglePlayServices.Awareness",
         "dependencyOnly": false
       },
       {
         "groupId": "com.google.android.gms",
         "artifactId": "play-services-base",
-        "version": "18.4.0",
-        "nugetVersion": "118.4.0",
+        "version": "18.5.0",
+        "nugetVersion": "18.5.0",
```

Comment:
Should be `118.5.0`, there are more than I have marked, they should all be fixed.